main.js

```diff
@@ -0,0 +1,306 @@
+var query = null;
+var todayFetch = function(query) {
+  lastQuery = encodeURIComponent(query);
+  $.ajax({
+    method: "GET",
+    url: "https://api.openweathermap.org/data/2.5/weather?q=" + lastQuery + "&APPID=3539d3ddceaed0d39c41bd28668ccd0e",
+    dataType: "json",
+    beforeSend: function() {
```

Comment:
It would be ideal to put "view" logic in a separate place than where you're doing your request